crates/nargo_cli/src/cli/prove_cmd.rs

```diff
@@ -122,7 +125,11 @@ pub(crate) fn prove_with_path<B: Backend, P: AsRef<Path>>(
     let (inputs_map, _) =
         read_inputs_from_file(&program_dir, prover_name.as_str(), Format::Toml, &abi)?;
 
-    let solved_witness = execute_program(backend, bytecode.clone(), &abi, &inputs_map)?;
+    let solved_witness = if let Some(driver) = context {
+        execute_program(backend, bytecode.clone(), &abi, &inputs_map, &debug.unwrap(), &driver)?
```

Comment:
Can we put the context and debug into the same Option so that we can remove the unwrap and make it clearer one is always Some when the other is?

Reply:
Done